docs/contributing.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to Authorino
+
+## Open issues
+
+Start by checking the list of [issues](https://github.com/3scale-labs/authorino/issues) in GitHub. Sometimes your contribution may have been filed in the past or part of another ongoing conversation.
+
+If you believe that your contribution is new, please open an issue before sending your pull-request.
```

Comment:
Do we want to say something like" open an issue, describe your idea and start a conversation"?

Reply:
I hope it's better now.